request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestController.java

```diff
@@ -59,7 +59,7 @@
  */
 public class RequestController implements Service<RequestController>, ServerActivity {
 
-    public static final ServiceName SERVICE_NAME = ServiceName.JBOSS.append("server", "global-request-controller");
+    public static final ServiceName SERVICE_NAME = RequestControllerRootDefinition.REQUEST_CONTROLLER_CAPABILITY.getCapabilityServiceName();
```

Comment:
Why? Service names are internal, and anyone referencing the request controller service name should have done it through the SERVICE_NAME constant anyway. If we keep the old name it will not be used anywhere.